api/db-cdr/generate-cdr/cb-criteria-refactor/debug/verify-domain-type-counts.sql

```diff
@@ -0,0 +1,895 @@
+-- ====== A COPY OF `all-of-us-ehr-dev.test_R2019q4r3` in `all-of-us-ehr-dev.ChenchalDummySrc` is used to VERIFY ======
```

Comment:
I'm uncomfortable with the hardcoded projects and datasets in here, but since it's only used as example SQL it's probably fine.